web_ui/src/pages/create-project/components/pose-template/readonly-template-manager.component.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+import { useEffect, useRef, useState } from 'react';
+
+import { Flex } from '@geti/ui';
+import { noop } from 'lodash-es';
+
+import { HoveredProvider } from '../../../../providers/hovered-provider/hovered-provider.component';
+import { SelectedProvider } from '../../../../providers/selected-provider/selected-provider.component';
+import { denormalizePoint } from '../../../../shared/utils';
+import useUndoRedoState from '../../../annotator/tools/undo-redo/use-undo-redo-state';
+import { ZoomProvider } from '../../../annotator/zoom/zoom-provider.component';
+import { TransformZoom } from '../../../shared/zoom/transform-zoom.component';
+import { CanvasTemplate } from './canvas/canvas-template.component';
+import { createRoi, resizePoints, TemplateState } from './util';
+
+export interface ReadonlyTemplateManagerProps {
+    className: string;
+    scaleFactor?: number;
+    initialNormalizedState: TemplateState;
+}
+
+export const ReadonlyTemplateManager = ({
```

Comment:
would it be better to have a little duplication but making a simpler template manager without any dependencies?

Reply:
Yes, a simpler version. Right now, we only need to render the template, not update it (at least for now; that could become a feature requirement later on).